fission/src/aps/APS.ts

```diff
@@ -122,12 +123,43 @@ class APS {
     }
 
     /**
-     * Logs the user out by setting their auth data to undefined.
+     * Logs the user out by setting their auth data to undefined and revoking their auth token.
+     * Does nothing if the auth token couldn't be revoked
```

Comment:
Should this comment line be removed now that we always set the auth token to undefined even if we can't revoke the token?

```suggestion
```